api/python/properties_prediction_d/datasets.py

```diff
@@ -0,0 +1,53 @@
+import config
+import pandas as pd  # type: ignore
+from dgl.data import DGLDataset  # type: ignore
+from dgl.data.utils import split_dataset  # type: ignore
+from dgl.dataloading import GraphDataLoader  # type: ignore
+from mol_to_graph import smiles_to_graph
```

Comment:
We need to import first-party modules (e.g. our own code) after third-party imports, and use this approach:

```python
from indigo.ml.mpp.mol_to_graph import smiles_to_graph
```